README.md

```diff
@@ -157,19 +164,14 @@ Type=oneshot
 ExecStart=/usr/bin/touch /tmp/file
 EOT
 
-if versioncmp($facts['puppetversion'],'6.1.0') < 0 {
-  include systemd::systemctl::daemon_reload
-  Systemd::Timer['daily.timer'] ~> Class[Systemd::Systemctl::Daemon_reload]
-}
-
 systemd::timer{'daily.timer':
   timer_content   => $_timer,
   service_content => $_service,
 }
 
 service{'daily.timer':
   ensure => running,
-  require => Systemd::Timer['daily.timer'],
+  notify => Systemd::Timer['daily.timer'],
```

Comment:
??? What does this do?

Reply:
I think it must do time travel.

Changed to subscribe - when the unit changes restart the service.